pages/subscribe/subscribe.js

```diff
@@ -77,8 +77,17 @@ async function main() {
         siteUrl
       });
       alert(I18N.getMessage("subscribe_subscribed", folderTitle));
+      
+      document.querySelector("#subscribe-button").disabled = true;
     });
 
+    // Disable Subscribe Button if the Feed is already in the Folder
+    const parentId = await Settings.getDefaultFolder();
+    const bookmarks = (await browser.bookmarks.search({title: title})).filter(node => node.parentId === parentId);
```

Comment:
Using the title for matching the feed doesn't seem very reliable. I think looking up the feed's url would be better.

Reply:
True, changed it to feed url